lldb/test/Shell/SwiftREPL/InitFile.test

```diff
@@ -0,0 +1,15 @@
+// Test that the Swift REPL init file works.
+// REQUIRES: system-darwin
+
+// RUN: if [ -f ~/.lldbinit ]; then mv ~/.lldbinit ~/.lldbinit.old; fi
```

Comment:
If the test fails it'll leave the lldbinit file in the old position. You can use `env HOME=%t` or something to spoof the home directory, we have other tests on llvm.org that do this too.